### ISSUE TITLE
__RecipeBook__/scripts/gui/info/list-box.lua:84: bad argument #1 of 2 to 'lower' (string expected, got nil)

### ISSUE THREAD

**Issue by PeteyPii** [raiguard/RecipeBook#166]
I get this error when clicking on `Crushed Vulcanite` in the recipe book when playing with SE and some other mods:

```
The mod Recipe Book (3.5.8) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event RecipeBook::on_gui_click (ID 1)
__RecipeBook__/scripts/gui/info/list-box.lua:84: bad argument #1 of 2 to 'lower' (string expected, got nil)
stack traceback:
	[C]: in function 'lower'
	__RecipeBook__/scripts/gui/info/list-box.lua:84: in function 'update'
	__RecipeBook__/scripts/gui/info/index.lua:328: in function 'update_contents'
	__RecipeBook__/control.lua:41: in function 'OPEN_PAGE'
	__RecipeBook__/scripts/gui/search/actions.lua:358: in function '?'
	__RecipeBook__/scripts/gui/search/index.lua:41: in function 'dispatch'
	__RecipeBook__/control.lua:298: in function 'handle_gui_action'
	__RecipeBook__/control.lua:305: in function 'read_gui_action'
	__RecipeBook__/control.lua:315: in function <__RecipeBook__/control.lua:313>
```

**Comment by raiguard** [comment]
Were you playing multiplayer?

**Comment by raiguard** [comment]
I have FINALLY reproduced this issue, after many months of not being able to!

**Comment by raiguard** [comment]
Fixed in flib 0.14.2, please update the `Factorio Library` mod to get the fix.
